azure-pipelines-v3.yml

```diff
@@ -43,10 +43,10 @@ jobs:
     displayName: dotnet test
     inputs:
       command: test
-      arguments: -c Release /p:CollectCoverage=true /p:CoverletOutputFormat=opencover"
+      arguments: -c Release
```

Comment:
We can probably use this task to upload code coverage on windows:

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-code-coverage-results?view=azure-devops

Reply:
I'm not following.
Window's reports are in Azure machine, how can GH action reach those?
You mean run windows UT with GH action too? How to manage the secrets then? Is keeping them in GH compliant?

Reply:
I mean run windows tests on azure pipeline and upload coverage data to azure pipeline, because codecov on mac does not cover certain test cases, see files with dropped coverage report. This is a low pri ask though.

Reply:
![image](https://user-images.githubusercontent.com/42199316/109763137-a09d4a00-7c2c-11eb-82af-cd2c73b2c95c.png)
working !